New context pad UI

smbea opened the pull request:
First part of https://github.com/camunda/improved-canvas/issues/5. It updates the context pad UI only (append feature and color picker + align/distribute menus next). Since we can release whenever/include only relevant modules, I believe it's easier to implement/review step by step. 

Note: I tried to re-use the existing context pad to the max so we avoid having to maintain 2 mostly duplicated context pads. So I overrode/added some things on top in order to work for this use case. Open to hearing other suggestions too.

philippfromme commented:
The color picker seems to have a couple of issues:

![image](https://github.com/camunda/improved-canvas/assets/7633572/b34c881d-2325-4b60-99e2-044554b0cba4)

* no border radius
* vertical alignment of the icons

Otherwise, this looks great! ⭐

smbea commented:
I actually meant to take care os the color picker and align/distribute menus next. The positioning also needs to be tweaked. But some of the color pick changes I was working on sneakily made it here. I removed them here so I can properly take care of it next. You can see how I split the context pad here: https://github.com/camunda/improved-canvas/issues/5

philippfromme reviewed (approved):
✅
